.gitignore

```diff
@@ -0,0 +1,5 @@
+.*.swo
+.*.swp
+
+server_standalone/server_standalone
```

Comment:
you can probably delete this as well, this would only happen if you did go build in the server_standalone directory